extensions/elixir/languages/elixir/tasks.json

```diff
@@ -11,9 +11,9 @@
     "args": ["test", "--failed"]
   },
   {
-    "label": "mix test $ZED_SYMBOL",
+    "label": "mix test $ZED_FILE",
     "command": "mix",
-    "args": ["test", "$ZED_SYMBOL"]
+    "args": ["test", "$ZED_FILE"]
```

Comment:
I wonder if we could have another variable, something like `ZED_RELATIVE_FILE_PATH`. The reason is that when running this task within umbrella apps, that will output test runs for all projects even though they are not run at all. For instance:

```
❯ mix test /Users/feliperenan/source/my_umbrella_app/apps/my_app/test/my_app_web/controllers/my_controller_test.exs
==> dice
......

Finished in 1.9 seconds (1.9s async, 0.00s sync)
6 tests, 0 failures

Randomized with seed 45266
==> another_app_within_umbrella_app


Finished in 0.00 seconds (0.00s async, 0.00s sync)
0 failures
```

While, if we use the relative path, it would be:

```
❯ mix test apps/my_app/test/my_app_web/controllers/my_controller_test.exs
==> my_app
......

Finished in 1.3 seconds (1.3s async, 0.00s sync)
6 tests, 0 failures

Randomized with seed 507991
```